runtime/onert/core/include/ir/operation/Reshape.h

```diff
@@ -37,12 +37,25 @@ class Reshape : public Operation
     SHAPE = 1
   };
 
+  struct Param
+  {
+    // Please see tensorflow/lite/c/builtin_op_data.h and reshape.cc.
+    // tensorflow lite supports only for ndim <= 8.
+    int shape[8];
+    int num_dimensions;
```

Comment:
How about change these param to a `std::vector`?

Reply:
@ragmani I just copied that code from TFLite, and they use above structure because they can't use dynamic data in Params. I agree with your opinion that it is good to use `std::vector`. I'll update it soon.